migrations/V192__Guidance_categories.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TYPE guidance_recommendation_category_type AS ENUM (
```

Comment:
can put this in V191 if preferred

Reply:
I'd prefer this stays in 192 -- it helps keep our migrations discrete (and there's really no cost to having 2 migration files over 1 unless they're doing very similar things!)

This way, it's:
- V191: Rename a bunch of stuff
- V192: Introduce a new feature (guidance categories)